service/src/test/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderServiceTest.java

```diff
@@ -288,13 +290,21 @@ private CaseData createPopulatedCaseData(GeneratedOrderType type, LocalDate loca
                     .order(GeneratedOrder.builder()
                         .title("Example Title")
                         .details("Example details")
+                        .build())
```

Comment:
would these case builder options be nicer in private methods too or would you like to keep them in here? - getting bulky

Reply:
also already in a private method